MapEditor.py

```diff
@@ -188,6 +194,23 @@ def _get_paths(self):
             title="Select a directory to place the patch.")
         self.settings.write(self.paths)
 
+    # region public functions
+
+    def return_edit(self, event):
+        self._edit_map()
+
+    def return_decompile(self, event):
+        self._decompile()
+
+    def tab_switch_dst(self, event):
+        self.dst_tv.focus_set()
+        return 'break'
+
+    def tab_switch_src(self, event):
+        self.src_tv.focus_set()
+        return 'break'
```

Comment:
why return `'break'`?

Reply:
Tab has some built-in functionality. [This Stackoverflow answer](https://stackoverflow.com/questions/4090683/overriding-default-tab-behaviour-in-python-tkinter) fixed it. That was my implementation of it.

Reply:
huh... TIL